src/client/sky.cpp

```diff
@@ -700,69 +570,198 @@ void Sky::update(float time_of_day, float time_brightness,
+				pointcolor_sun_f.r =
+						pointcolor_light *
+						(float)m_materials[3]
+								.EmissiveColor.getRed() /
+						255;
+				pointcolor_sun_f.b =
+						pointcolor_light *
+						(float)m_materials[3]
+								.EmissiveColor.getBlue() /
+						255;
 				pointcolor_sun_f.g = pointcolor_light *
-					(float)m_materials[3].EmissiveColor.getGreen() / 255;
+						     (float)m_materials[3]
+								     .EmissiveColor
+								     .getGreen() /
+						     255;
 			} else {
 				pointcolor_sun_f.r = pointcolor_light * 1;
-				pointcolor_sun_f.b = pointcolor_light *
-					(0.25 + (rangelim(m_time_brightness, 0.25, 0.75) - 0.25) * 2 * 0.75);
-				pointcolor_sun_f.g = pointcolor_light * (pointcolor_sun_f.b * 0.375 +
-					(rangelim(m_time_brightness, 0.05, 0.15) - 0.05) * 10 * 0.625);
+				pointcolor_sun_f.b =
+						pointcolor_light *
+						(0.25 + (rangelim(m_time_brightness, 0.25,
+									 0.75) -
+									0.25) * 2 *
+										0.75);
+				pointcolor_sun_f.g =
+						pointcolor_light *
+						(pointcolor_sun_f.b * 0.375 +
+								(rangelim(m_time_brightness,
+										 0.05,
+										 0.15) -
+										0.05) *
+										10 *
+										0.625);
 			}
 
 			video::SColorf pointcolor_moon_f(0.5 * pointcolor_light,
-				0.6 * pointcolor_light, 0.8 * pointcolor_light, 1);
+					0.6 * pointcolor_light, 0.8 * pointcolor_light,
+					1);
 			if (m_moon_tonemap) {
-				pointcolor_moon_f.r = pointcolor_light *
-					(float)m_materials[4].EmissiveColor.getRed() / 255;
-				pointcolor_moon_f.b = pointcolor_light *
-					(float)m_materials[4].EmissiveColor.getBlue() / 255;
+				pointcolor_moon_f.r =
+						pointcolor_light *
+						(float)m_materials[4]
+								.EmissiveColor.getRed() /
+						255;
+				pointcolor_moon_f.b =
+						pointcolor_light *
+						(float)m_materials[4]
+								.EmissiveColor.getBlue() /
+						255;
 				pointcolor_moon_f.g = pointcolor_light *
-					(float)m_materials[4].EmissiveColor.getGreen() / 255;
+						      (float)m_materials[4]
+								      .EmissiveColor
+								      .getGreen() /
+						      255;
 			}
 
 			video::SColor pointcolor_sun = pointcolor_sun_f.toSColor();
 			video::SColor pointcolor_moon = pointcolor_moon_f.toSColor();
 			// Calculate the blend color
-			pointcolor = m_mix_scolor(pointcolor_moon, pointcolor_sun, pointcolor_blend);
+			pointcolor = m_mix_scolor(pointcolor_moon, pointcolor_sun,
+					pointcolor_blend);
 		}
 		m_bgcolor = m_mix_scolor(m_bgcolor, pointcolor, m_horizon_blend() * 0.5);
-		m_skycolor = m_mix_scolor(m_skycolor, pointcolor, m_horizon_blend() * 0.25);
+		m_skycolor = m_mix_scolor(
+				m_skycolor, pointcolor, m_horizon_blend() * 0.25);
 	}
 
 	float cloud_direct_brightness = 0.0f;
 	if (sunlight_seen) {
 		if (!m_directional_colored_fog) {
 			cloud_direct_brightness = time_brightness;
-			// Boost cloud brightness relative to sky, at dawn, dusk and at night
+			// Boost cloud brightness relative to sky, at dawn, dusk and at
+			// night
```

Comment:
Please only wrap lines at 90 columns (tab size 4). This may be a bad clang format suggestion.